DigitalOffice/Tests/UserServiceTests/Mappers/InternalUserInfoResponseMapperTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using Broker.Responses;
+using LT.DigitalOffice.UserService.Database.Entities;
+using LT.DigitalOffice.UserService.Mappers;
+using LT.DigitalOffice.UserService.Mappers.Interfaces;
+using LT.DigitalOffice.UserServiceUnitTests.UnitTestLibrary;
+using NUnit.Framework;
+
+namespace LT.DigitalOffice.UserServiceUnitTests.Mappers
+{
+    public class InheritedDerInternalUserPositionResponse : InternalUserPositionResponse
+    {
+        public InheritedDerInternalUserPositionResponse(string userPositionName)
+        {
+            this.UserPositionName = userPositionName;
+        }
+
+        public string UserPositionName { get; }
+    }
+
+    public class InternalUserInfoResponseMapperTests
+    {
+        private DbUser dbUser;
+        private InternalUserPositionResponse userPosition;
+        private ITwoModelsMapper<DbUser,InternalUserPositionResponse, object> mapper;
+
+        private readonly Guid id = Guid.NewGuid();
+        private readonly string firstName = "Ivan";
+        private readonly string lastName = "Ivanov";
+        private readonly string middleName = "Ivanovich";
+        private readonly string position = "Software Engineer";
+
+        
+        [SetUp]
+        public void Setup()
+        {
+            mapper = new InternalUserInfoResponseMapper();
+        }
+
+        [Test]
+        public void ThrowExceptionIfArgumentIsNull()
+        {
+            Assert.Throws<NullReferenceException>(() => mapper.Map(null,null));
```

Comment:
Why not throw ArgumentNullException from mapper? This will give more information.

Reply:
I think that this is not so important, because we will replace these exceptions with our custom exceptions, but I fixed it